static/js/lib/site_content.ts

```diff
@@ -38,13 +41,13 @@ const isMainContentField = (field: ConfigField) =>
  * Translates page content form values into a payload that our REST API understands.
  */
 export const contentFormValuesToPayload = (
-  values: { [key: string]: string | File },
+  values: Record<string, any>,
```

Comment:
is there any way we can avoid `any` here?

Reply:
I changed it to `Record<string, string | File>` and didn't get any type errors except for one in a test file